robot_name: alfie
dynamics:
  alfie:
    mass: 20
    buoyancy: 21
    center_of_gravity:
      x: 0
      y: 0
      z: -0.1
    center_of_buoyancy:
      x: 0
      y: 0
      z: 0.1
    added_mass:
      x: 0
      y: 0
      z: 0
      k: 0
      m: 0
      n: 0
    inertia:
      ixx: 10
      ixy: 0
      ixz: 0
      iyy: 10
      iyz: 0
      izz: 10
    linear_damping_coeffs:
      x: 0
      y: 0
      z: 0
      k: 0
      m: 0
      n: 0
    rate: 50
